### ISSUE TITLE
anon72.yaml and hexherhide666.yaml point to the same user

### ISSUE THREAD

**Issue by taiju** [rms-support-letter/rms-support-letter.github.io#4540]
```shell
$ find _data/signed -type f -iname *.yaml | xargs md5sum | sort | awk '{if (a[$1]) { print $1, a[$1], $2 } else {a[$1]=$2}}'
7b800a5c1e5122b5dc2f4036d2b7db6c _data/signed/anon72.yaml _data/signed/hexherhide666.yaml
```
Deletions and additions seem to be alternating.
https://github.com/rms-support-letter/rms-support-letter.github.io/commits/master/_data/signed/anon72.yaml

rel: https://github.com/rms-support-letter/rms-support-letter.github.io/issues/4134

![image](https://user-images.githubusercontent.com/95721/112859906-450c9200-90ee-11eb-813b-f659ed4313eb.png)

**Comment by shenlebantongying** [comment]
Those two are behind a real mental illness, will delete soon,
https://github.com/rms-support-letter/rms-support-letter.github.io/issues/4134

**Comment by taiju** [comment]
🙈 
I understood. close it.

**Comment by shenlebantongying** [comment]
Can you use your script to identify some 1 day accounts?

**Comment by taiju** [comment]
I'd like to try (and contribute), but I don't think I have the time to spend working on it. I apologize :bow:
And since my time zone is Asia/Tokyo, I'm going to bed 🛌

**Comment by shenlebantongying** [comment]
Ok, thanks

**Comment by taiju** [comment]
@shenlebantongying It was a little messy, but I tried.

```shell
find _data/signed -type f -iname '*.yaml' \
    | xargs grep -P '\Alink:\s+https://github.com/' \
    | sed 's|_data/signed/.*\.yaml:link: https://github.com/||;s|/.*||;s|\r||' \
    | xargs -n1 -P4 -i curl -I -s -o /dev/null -w 'url: %{url_effective}, http_code: %{http_code}\n' -u <username>:<access_token> https://api.github.com/users/{} \
    | grep -v 'http_code: 200'
url: https://api.github.com/users/arthureroberer, http_code: 404
url: https://api.github.com/users/x3m1lx, http_code: 404
url: https://api.github.com/users/gwofford11, http_code: 404
url: https://api.github.com/users/homeless-schizo-retard, http_code: 404
url: https://api.github.com/users/orgs, http_code: 404
url: https://api.github.com/users/isu29, http_code: 404
url: https://api.github.com/users/RadCatGirl, http_code: 404
```

The script was not good and detected the Organization, but if I remove it manually, the following users do not already exist.

- arthureroberer
- x3m1lx
- gwofford11
- homeless-schizo-retard
- isu29
- RadCatGirl

However, this only applies to users who have included the GitHub URL in their `link`.

**Comment by shenlebantongying** [comment]
Removed 1
Corrected 4
arthureroberer https://github.com/rms-support-letter/rms-support-letter.github.io/commit/283e9622d044a0c744d3cb216d087046f7d813e4
x3m1lx https://github.com/rms-support-letter/rms-support-letter.github.io/commit/7c20649b6ebaac5a0434464f38caea04512ad36f
 gwofford11 https://github.com/rms-support-letter/rms-support-letter.github.io/commit/7937ca4eda1fcb8a663a307d0683c2ae61dff78c
 homeless-schizo-retard https://github.com/rms-support-letter/rms-support-letter.github.io/commit/1ac019671703fa089ff9e0ea58af4ad8fe721c33
 RadCatGirl https://github.com/rms-support-letter/rms-support-letter.github.io/commit/a0757cbc286ea02cc0f37cb17c8bf24ee2503486

This one should be real
isu29

**Comment by taiju** [comment]
The rate limit of the API is 5,000 per hour, so it cannot run for a while. Also I am having difficulty running it today.

> This one should be real
> isu29

Is it real?
https://github.com/isu29

**Comment by shenlebantongying** [comment]
She/He seems deleted his account 

![image](https://user-images.githubusercontent.com/20123683/112880534-8ffbba80-9098-11eb-89cd-5b915ee2e5a0.png)
![image](https://user-images.githubusercontent.com/20123683/112880586-9c801300-9098-11eb-8c5f-4e8042d1a12d.png)

https://github.com/rms-support-letter/rms-support-letter.github.io/blob/123f542a4f14deace603250dd4629d3b9b29ca97/_data/signed/isu29.yaml

https://github.com/rms-support-letter/rms-support-letter.github.io/commit/7f2c6302c9d4779603e19a4bb416db697d14c57f

**Comment by shenlebantongying** [comment]
I deleted it.

**Comment by taiju** [comment]
We can't do this today, but we can apply this to identify users who have created GitHub accounts within a day.

As the number of signatures increases, the probability of exceeding the API limit increases, so it is better to use a webhook and inspect the PR as it is received.

Thank you for your work.

**Comment by shenlebantongying** [comment]
If you don't time, you don't have to. Thanks!

**Comment by shenlebantongying** [comment]
@taiju Can you rerun it to check new users?

**Comment by taiju** [comment]
All right. I'll check it later.

**Comment by taiju** [comment]
@shenlebantongying  I checked the following procedure.

1. Extract usernames

    ```shell
    $ find _data/signed -type f -iname '*.yaml' | xargs grep -P '\Alink:\s+https://github.com/' | sed 's|_data/signed/.*\.yaml:link: https://github.com/||' > usernames
    $ wc -l username
    4323
    ```

2. Manually adjust the data

    1. Remove trailing slash
    2. Remove control characters (`r-novel`)
    3. Exclude organization accounts (`orgs/iGNUranza`)
    4. etc ('ChirrutDataRebel/ChirrutDataRebel.github.io` => `ChirrutDataRebel`)

3. Check user existence (part 1) (Split steps to avoid API limitations)

    ```shell
    $ cat usernames | head -n2500 | xargs -n1 -P4 -i curl -I -s -o /dev/null -w 'url: %{url_effective}, http_code: %{http_code}\n' -u <username>:<token> https://api.github.com/users/{} | grep -v 'http_code: 200'
    url: https://api.github.com/users/recepsaglam, http_code: 404
    url: https://api.github.com/users/ShitPissCum1312, http_code: 404
    url: https://api.github.com/users/elonmusksama, http_code: 404
    url: https://api.github.com/users/Meemslh, http_code: 404
    url: https://api.github.com/users/Sagandh, http_code: 404
    url: https://api.github.com/users/UN5P0K3N, http_code: 404
    url: https://api.github.com/users/Lux-OpenSrc, http_code: 404
    url: https://api.github.com/users/UN5P0K3N, http_code: 404
    url: https://api.github.com/users/pelot99, http_code: 404
    url: https://api.github.com/users/javadnikbakh, http_code: 404
    url: https://api.github.com/users/LutsenkoKirill, http_code: 404
    ```

4. Check user existence (part 2)

    ```shell
    $ cat usernames | tail -n+2501 | xargs -n1 -P4 -i curl -I -s -o /dev/null -w 'url: %{url_effective}, http_code: %{http_code}\n' -u <username>:<token> https://api.github.com/users/{} | grep -v 'http_code: 200'
    url: https://api.github.com/users/HexaOneOfficial, http_code: 404
    url: https://api.github.com/users/6opoDuJIo, http_code: 404
    url: https://api.github.com/users/1-methyl-2-phenyl-ethylamine, http_code: 404
    url: https://api.github.com/users/crkvt, http_code: 404
    url: https://api.github.com/users/KamilKowalski157, http_code: 404
    url: https://api.github.com/users/pzenov, http_code: 404
    url: https://api.github.com/users/ApolloDev0, http_code: 404
    ```

In summary, the following users did not exist.

1. recepsaglam
1. ShitPissCum1312
1. elonmusksama
1. Meemslh
1. Sagandh
1. UN5P0K3N
1. Lux-OpenSrc
1. UN5P0K3N
1. pelot99
1. javadnikbakh
1. LutsenkoKirill
1. HexaOneOfficial
1. 6opoDuJIo
1. 1-methyl-2-phenyl-ethylamine
1. crkvt
1. KamilKowalski157
1. pzenov
1. ApolloDev0

**Comment by shenlebantongying** [comment]
Thanks a lot!

I will check them one by one later.

**Comment by taiju** [comment]
You're welcome. Thank you for everything!

**Comment by taiju** [comment]
I forgot to tell you. The commit ID of the source I checked is eed7e5f2.

**Comment by shenlebantongying** [comment]
They are removed at https://github.com/rms-support-letter/rms-support-letter.github.io/commit/537bfb0cccee0bd39ba338a030288e27afed53da
